pkg/tests/api/dashboards/api_dashboards_test.go

```diff
@@ -33,28 +32,13 @@ func TestMain(m *testing.M) {
 }
 
 func TestIntegrationDashboardQuota(t *testing.T) {
-	if testing.Short() {
```

Comment:
These tests were testing both with the feature flag on, and with them off. We can now just do one again 🎉